main/main_gc-menu2.cpp

```diff
@@ -341,18 +347,33 @@ int main(int argc, char* argv[]) {
 
 #ifdef HW_RVL
 	load_config(&argv[0][0]);
-	// Handle options passed in through arguments
-	int i;
-	for(i=1; i<argc; ++i){
-		handleConfigPair(argv[i]);
+	if(argc != 0 && argv != 0)
+	{
+		load_config(&argv[0][0]);
+		// Handle options passed in through arguments
+		int i;
+		for(i=1; i<argc; ++i){
+			handleConfigPair(argv[i]);
+		}
 	}
-#else
-	load_config("sd");
+	else
 #endif
-	//Switch to MiniMenu if active
+		load_config("sd");
+
 	if (miniMenuActive)
-	{
+
 		menu->setUseMiniMenu(true);
+	
+	if (menu->Autoboot)
+	{
+		// after init wpad wait a bit
+		sleep(2);

```

Comment:
I noticed this also came up in Not64. I'd rather spin with a message box here until there's a controller detected by the menu, I can have a look at adding that though.

Reply:
i've noticed Extrems already fixed that here, so this sleep function would be useless anyway: https://github.com/extremscorner/not64/commit/745a66a3779919bf309beea56257b568bb645f22#diff-fa6eef4498a7132fd1f6aebfe3124f4be1441daf9081b91ad105850f41abc985

Reply:
I've removed the sleep function in https://github.com/emukidid/Wii64/pull/98/commits/d3395d82faba9bdd0231bf7729878102e7ea40c9, cause this commit from Not64 fixes controllers when in autoboot anyway: https://github.com/extremscorner/not64/commit/745a66a3779919bf309beea56257b568bb645f22#diff-515d9a7589dc16cc8b4da22fdd8bba23732a67fa26bb63081149dcdacfc64e1c